acyclic, bcomps, ccomps, edgepaint, gvcolor, gvpack, mingle, nop, sccmap, unflatten: add page (#6401)
tldr-pages/tldr: pages/common/edgepaint.md

# edgepaint

> Colorize edges of a graph layout to clarify crossing edges.
> Graphviz filters: `acyclic`, `bcomps`, `comps`, `edgepaint`, `gvcolor`, `gvpack`, `mingle`, `nop`, `sccmap`, `tred`, & `unflatten`.
> More information: <https://graphviz.org/pdf/edgepaint.1.pdf>.

- Colorize edges of one or more graph layouts (that already have layout information) to clarify crossing edges:

`edgepaint {{path/to/layout1.gv}} {{path/to/layout2.gv ...}} > {{path/to/output.gv}}`

- Colorize edges using a color scheme. (See <https://graphviz.org/doc/info/colors.html#brewer>):

`edgepaint -color-scheme={{accent7}} {{path/to/layout.gv}} > {{path/to/output.gv}}`

> - Perform layout, colorization, and output to a picture with one command:

Reviewer: @sbrl what do you think about this formulation? 
Still not perfect, but maybe an improvement:
```suggestion
- Perform layouting, colorization, and rendering to an image with one command:
```
I don't think `layouting` is a word though.
